### PR TITLE
Small refactor

### PR DESCRIPTION
**Pull request by BeeHiveJava**

### CODE REVIEW

**Comment by BeeHiveJava** [comment]
test this please

**Comment by BeeHiveJava** [comment]
retest this please

File: data/src/main/java/nl/tehdreamteam/se42/data/conversation/hibernate/ConversationHibernateRepository.java [hunk 1]
```diff
@@ -22,34 +21,22 @@ protected ConversationDAO createDao(EntityManager manager) {
...
 
     @Override
     public void remove(long id) {
-        Conversation conversation = find(id);
+        Conversation conversation = get(id);
         remove(conversation);
     }
 
     @Override
     public void remove(Conversation conversation) {
```
Comment (ghost) [inline data/src/main/java/nl/tehdreamteam/se42/data/conversation/hibernate/ConversationHibernateRepository.java:39]: MINOR Remove this method to simply inherit it. [rule](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1185)

**Comment by ghost** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Pull request merged**